### PR TITLE
Show document count for document listing views

### PR DESCRIPTION
**Pull request by nickmwangemi**
This PR adds functionality for showing document count for the various document listing views
Demo for Legal Instruments, Judgments, Soft Law, Reports & Guides
https://www.loom.com/share/a8221bd2000042a9980bca26e60fdfdc

Demo for Topics:
https://www.loom.com/share/0467322d6ff944608c61f23eed8168de

### CODE REVIEW

**Comment by longhotsummer** [comment]
@nickmwangemi nice!

Just a thought -- do you think the document count would be better a bit smaller, maybe just normal text size? It's supplementary information, after all.

**Comment by nickmwangemi** [comment]
> @nickmwangemi nice!
> 
> Just a thought -- do you think the document count would be better a bit smaller, maybe just normal text size? It's supplementary information, after all.

Makes sense. Lemme change that.

File: peachjam/templates/peachjam/court_detail.html [hunk 1]
```diff
@@ -23,7 +23,8 @@
   {% endwith %}
 {% endblock %}
 {% block page-header %}
-  <h1 class="my-4">{{ formatted_court_name }}</h1>
+  <h1 class="my-4 mb-2">{{ formatted_court_name }}</h1>
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/court_detail.html:26]:
```suggestion
  <h1 class="mt-4">{{ formatted_court_name }}</h1>
```

File: peachjam/templates/peachjam/legal_instrument_list.html [hunk 2]
```diff
@@ -4,5 +4,6 @@
   {% trans 'Legal Instruments' %}
 {% endblock %}
 {% block page-title %}
-  <h1 class="my-4">{% trans 'Legal Instruments' %}</h1>
+  <h1 class="my-4 mb-2">{% trans 'Legal Instruments' %}</h1>
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/legal_instrument_list.html:7]:
```suggestion
  <h1 class="mt-4">{% trans 'Legal Instruments' %}</h1>
```

File: africanlii/templates/peachjam/soft_law_list.html [hunk 3]
```diff
@@ -4,5 +4,6 @@
   {% trans 'Soft Law' %}
 {% endblock %}
 {% block page-title %}
-  <h1 class="my-4">{% trans 'Soft Law' %}</h1>
+  <h1 class="my-4 mb-2">{% trans 'Soft Law' %}</h1>
```
Comment (longhotsummer) [inline africanlii/templates/peachjam/soft_law_list.html:7]:
```suggestion
  <h1 class="mt-4">{% trans 'Soft Law' %}</h1>
```

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**